### PR TITLE
DOCSP-40287: Read landing page

### PR DESCRIPTION
**Pull request by mcmorisi**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-scala/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-40287>
Staging - <https://deploy-preview-85--docs-scala.netlify.app/read/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/includes/usage-examples/SampleReadApp.scala [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import org.mongodb.scala.model.Filters._
+import org.mongodb.scala.model.changestream._
+
+object SampleReadApp {
+
+  def main(args: Array[String]): Unit = {
+    val mongoClient = MongoClient("<connection string URI>")
+    val database: MongoDatabase = mongoClient.getDatabase("<database name>")
+    val collection: MongoCollection[Document] = database.getCollection("<collection name>")
+
```
Comment (norareidy) [inline source/includes/usage-examples/SampleReadApp.scala:13]: S: you can delete this extra newline

File: source/read.txt [hunk 2]
```diff
@@ -30,5 +30,136 @@ Read Data from MongoDB
...
+given filter:
+
+.. literalinclude:: /includes/usage-examples/ReadCodeExamples.scala
+   :start-after: start-find-one
+   :end-before: end-find-one
+   :language: scala
+   :copyable:
+   :dedent:
+
+To learn more about the ``find().first()`` method, see the :ref:`Retrieve Data <scala-retrieve>` guide.
```
Comment (norareidy) [inline source/read.txt:75]: S: I'm not sure if it's accurate to say `find().first() `method, maybe just `first()` - or "learn more about chaining the first() method to the find() method"

```suggestion
To learn more about the ``first()`` method, see the :ref:`Retrieve Data <scala-retrieve>` guide.
```

File: source/read.txt [hunk 3]
```diff
@@ -30,5 +30,136 @@ Read Data from MongoDB
...
+   :dedent:
+
+To learn more about the ``countDocuments()`` method, see the :ref:`Count Documents <scala-count>` guide.
+
+Count Documents Returned from a Query
+-------------------------------------
+
+The following example returns the number of documents in the specified
+collection that match the criteria specified by
+the given filter:
```
Comment (norareidy) [inline source/read.txt:111]: S: just to avoid repeating "specified" and make this more concise, you can just say "query criteria"
```suggestion
collection that match the query criteria:
```

File: source/read.txt [hunk 4]
```diff
@@ -30,5 +30,136 @@ Read Data from MongoDB
    /read/count
    /read/change-streams
 
+Overview
+--------
 
+On this page, you can see copyable code examples that show common methods you
+can use to read data from MongoDB by using the {+driver-short+}.
```
Comment (norareidy) [inline source/read.txt:37]: S: I'd reword this to avoid repeating use/using

```suggestion
On this page, you can see copyable code examples that show common {+driver-short+} methods you
can use to read data from MongoDB.
```

**Review by norareidy — APPROVED** [review]
LGTM with minor suggestions

**Review by jyemin — APPROVED** [review]
